modules/core/include/opencv2/core.hpp

```diff
@@ -1616,6 +1616,7 @@ CV_EXPORTS_W bool checkRange(InputArray a, bool quiet = true, CV_OUT Point* pos
                             double minVal = -DBL_MAX, double maxVal = DBL_MAX);
 
 /** @brief converts NaN's to the given number
+@param a input/output matrix (CV_32FC1 type).
```

Comment:
I propose to replace `CV_32FC1` with `CV_32F`, amount of channels does not matter.

Reply:
Ok, fixed